### ISSUE TITLE
$header-sizes sass map error

### ISSUE THREAD

**Issue by jdhankins80** [JeremyEnglert/JointsWP#137]
Hey Jeremy,

When I run gulp I am getting an error when it runs across the $header-sizes map. Here is what I am getting:

[12:30:13] assets\scss\_settings.scss
Error: (small: ("h1": 24, "h2": 20, "h3": 19, "h4": 18, "h5": 17, "h6": 16), med
ium: ("h1": 48, "h2": 40, "h3": 31, "h4": 25, "h5": 20, "h6": 16)) isn't a valid
 CSS value.
        on line 101 of assets/scss/_settings.scss
     small: (
   --^

I have been updating my theme from F5 and was just getting starting on the scss. This is a fresh _settings file with no changes made yet.
I am fairly new to gulp so maybe I am missing a dependency???

**Comment by JeremyEnglert** [comment]
Hmmm. It looks like Gulp is having issues with the Sass map. I don't think it's a dependency problem. 

It looks like the settings file may need to be updated, the one in the newest version of Foundation looks slightly different: https://github.com/JeremyEnglert/JointsWP/blob/master/assets/scss/_settings.scss

**Comment by jdhankins80** [comment]
Still working on this, but I have tried updating to the latest settings file with the same result. I also tried compiling with Compass and it is throwing up a similar error...

Only other comment I've seen on this was here: http://foundation.zurb.com/forum/posts/36191-header-sizes-sass-error

I'll keep working on it and also try updating my CLI tools/ plugins

**Comment by JeremyEnglert** [comment]
@jdhankins80, I'm having a hard time replicating this as well.

Sass Maps are somewhat new to Sass - so if you're running an older version of Sass, they may throw errors. However, it should be using the correct dependencies are they are packaged with the theme - which makes this a real head scratcher. 

If you can, you should reply to the thread on the Foundation forum and let them know you're having the same issue. It may also be worth opening an issue on the foundation-sites GitHub page. I'm thinking this is related to Foundation somehow (and not specific to JointsWP). 

I'm going to keep this open here as well.

**Comment by jdhankins80** [comment]
Well I wasn't able to figure out what was going on, but I did get things working. I ended up starting fresh and creating an entirely new project directory, downloading the latest joints, and was able to compile everything on the first go. Then I started copying back in my original project files (overwriting) and compiling trying to narrow down the problem. In the end I was able to copy everything back in and never had any issues!? Looks like this may have just been some sort of glitch on my end so I think we can close this out.
